root/etc/s6-overlay/s6-rc.d/init-wireguard-confs/run

```diff
@@ -25,7 +30,7 @@ generate_confs () {
         wg genkey | tee /config/server/privatekey-server | wg pubkey > /config/server/publickey-server
     fi
     eval "$(printf %s)
-    cat <<DUDE > /config/wg0.conf
+    cat <<DUDE > /config/wg_confs/wg0.conf
```

Comment:
`/app/show-peer` also needs to be changed for the new path

Reply:
I remembered this last night on mobile :)